src/localization/translationProvider.ts

```diff
@@ -0,0 +1,18 @@
+let translation: any = null;
+
+export const L = (key: string): string => {
+  if (!translation) {
+    translation = getTranslation();
+  }
+  
+  return translation[key] ?? key;
+};
+
+const getTranslation = (): any => {
```

Comment:
We do know the type of this, we should use the proper type. [Here](https://www.typescriptlang.org/docs/handbook/2/typeof-types.html)'s how I think you could do this.